### PR TITLE
Update test projects to NServiceBus 7.1 and .NET Core 2.1

### PR DESCRIPTION
**Pull request by timbussmann**

### CODE REVIEW

**Comment by SeanFeldman** [comment]
Are we going to move forward with 7.1 or hold off for 7.2?

**Comment by timbussmann** [comment]
I'm not aware of any 7.2. There will be a 7.1.1 but there's no strict need unless the issue actually causing issues for this repo (which it isn't afaik)

File: src/TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj [hunk 1]
```diff
@@ -15,10 +15,10 @@
   <ItemGroup>
     <PackageReference Include="WindowsAzure.Storage" Version="8.*" />
     <!-- TODO: replace with "3.*" when NUnit bug is fixed -->
```
Comment (SeanFeldman) [inline src/TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: Shouldn't this comment be removed?

Comment (timbussmann) [reply src/TransportTests/NServiceBus.AzureStorageQueues.TransportTests.csproj]: yup. Done.

**Pull request merged**